parse.test.js

```diff
@@ -160,6 +160,20 @@ var tests = [
             {number: '', title: 'Artist 2 - Second track', time: '4:56'},
         ],
     },
+    {
+        str: '\`Leave The Door Open\`',
+        lineMode: 'one-line',
+        expected: [
+            {number: '', title: "'Leave The Door Open'", time: ''},
+        ],
+    },
+    {
+        str: '‘Leave The Door Open’',
+        lineMode: 'one-line',
+        expected: [
+            {number: '', title: "'Leave The Door Open'", time: ''},
+        ],
+    },
```

Comment:
We're missing double smart quotes (and the regex above):

```js
    {
         str: '“double smart quotes”',
         lineMode: 'one-line',
         expected: [
             {number: '', title: "\"double smart quotes\"", time: ''},
         ],
    },
```

Also, please rebase because I added a CI job: #23